build/deploy.sh

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env bash
 
-GETH_ARCHIVE_NAME="multi-geth-$TRAVIS_OS_NAME"
+GETH_ARCHIVE_NAME="ethoxy-multi-geth-$TRAVIS_OS_NAME"
```

Comment:
Note that I've added this prefix. I don't love modifying the already-established name of the build, but did so because it creates a consistent namespace for the archives... and makes for simpler artifact globbing during the CI deploy steps.

Reply:
... but it's fine with we me to not do this, too.

Reply:
@whilei Can we just use `multi-geth` as the prefix? Who knows tomorrow whether `multi-geth` will still be under `ethoxy` umbrella. :)

Reply:
Yea, of course. Is `ethoxy-alltools` alright with you, while I'm at it?

Reply:
@whilei How about `multi-geth-alltools`?